### PR TITLE
Single command per example

### PR DESCRIPTION
**Pull request by notpeter**
Gets almost all of #687.

### CODE REVIEW

**Comment by rubenvereecken** [comment]
First off before I dig in, I love you for this!

**Comment by notpeter** [comment]
I'm actually at fault for almost all of these multi-liners so cleaning up my mess is the least I could do.

**Comment by igorshubovych** [comment]
Wow. That's an amazing work. I always wanted to do this, but was always too lazy for it.

However in future it is better to split it by page. Imagine somebody will have ideas on how to improve it, you will have long discussion and we cannot merge the whole nice PR, just because of unfinished discussion about minor things.

**Comment by igorshubovych** [comment]
Actually how about we merge it sooner and then we can open PRs on individual pages in case of need?

**Comment by notpeter** [comment]
Well, I certainly wasn't going to create a dozen individual branches and pull requests. 

If there's issues I'm happy to cherry pick contentious commits into their own branches/PRs for discussion and rebase them off this branch.

File: pages/common/cal.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
 
-- Display a calendar for the current year or a specified year:
+- Display a calendar for a specific month:
+
+`cal -m {{month_number}}`
+
+- Display all 12 months of calendar for a specific year:
 
-`cal -y`
 `cal 2013`
```
Comment (waldyrious) [inline pages/common/cal.md]: While we're at it, if you don't mind, please change 2013 to {{year}}. And perhaps include a `cal -y` example before this one.

File: pages/common/ls.md [hunk 2]
```diff
@@ -2,33 +2,22 @@
...
+`ls -a`
 
-- List all files with their rights, groups, owner:
+- Long format list (permissions, ownership, size and modification date) of all files:
 
-`ls -l`
+`ls -la`
 
-- List all files and display the file size in a human readable format:
+- Long format list with size displayed using human readable units (KB,MB,GB):
```
Comment (waldyrious) [inline pages/common/ls.md]: Spaces after commas?

File: pages/common/rename.md [hunk 3]
```diff
@@ -2,19 +2,18 @@
...
 > Renames multiple files.
 
-- Change foo to bar in matching filenames:
+- Rename files using a Perl Common Regular Expression (substitute 'foo' with 'bar' wherever found):
 
-`rename {{'s/foo/bar/'}} {{*.txt}}`
+`rename {{'s/foo/bar/'}} {{\*}}`
 
-- Convert to lower case:
+- Dry-run, display which renames would occur without performing them:
```
Comment (waldyrious) [inline pages/common/rename.md]: I'd use a dash rather than a comma here.

File: pages/common/srm.md [hunk 4]
```diff
@@ -1,17 +1,19 @@
 # srm
 
 > Securely remove files or directories.
-> Overwrites the existing data one or multiple. Drop in replacement for rm.
+> Overwrites the existing data one or multiple times. Drop in replacement for rm.
 
-- Remove a file after overwriting (single pass, 7 pass, 35 pass):
+- Remove a file after a single pass overwriting with random data:
```
Comment (waldyrious) [inline pages/common/srm.md]: I'd suggest either "a single-pass overwriting" or "a single pass of overwriting"

File: pages/common/srm.md [hunk 5]
```diff
@@ -1,17 +1,19 @@
...
 
 `srm -s {{/path/to/file}}`
+
+- Remove a file after seven passes overwriting it with random data:
```
Comment (waldyrious) [inline pages/common/srm.md]: Same as above.

File: pages/common/srm.md [hunk 6]
```diff
@@ -1,17 +1,19 @@
...
+
 `srm -m {{/path/to/file}}`
-`srm {{/path/to/file}}`
 
-- Scurely remove recursively a directory and all it's subdirectories:
+- Recursively remove a directory and contents overwriting each file with a single pass of random data:
```
Comment (waldyrious) [inline pages/common/srm.md]: Suggestion: "and its contents,"

**Comment by waldyrious** [comment]
I've done a detailed review of the first few diffs, and added minor comments inline, which I assume are uncontroversial. I'll stop there, assuming the rest of the changes are in the same vein, to avoid cluttering this PR with too much nitpicking. You made very good calls IMO in choosing what examples to keep and rephrasing some descriptions.

**Comment by igorshubovych** [comment]
@waldyrious, it seems like @notpeter will reply only tomorrow. What if we merge it now, and you will create another PR with your changes?

**Comment by waldyrious** [comment]
@igorshubovych let's see what @notpeter has to say. IMO we can wait until tomorrow.

**Comment by notpeter** [comment]
@waldyrious All you suggestions improved clarity so I implemented them all.   Keep the comments coming.

**Comment by waldyrious** [comment]
Thanks @notpeter. I won't push it, though -- from what I've seen, I trust your judgment and am :+1: with merging this ASAP.

**Comment by igorshubovych** [comment]
@notpeter great thanks on taking responsibility and doing this! well done!

**Pull request merged**